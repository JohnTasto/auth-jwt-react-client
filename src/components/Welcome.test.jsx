/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import Welcome from './Welcome'

describe('<Welcome />', () => {
  test('Renders correctly', () => {
    const tree = renderer.create(<Welcome />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
