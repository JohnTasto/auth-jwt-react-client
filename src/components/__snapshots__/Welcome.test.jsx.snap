// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome /> Renders correctly 1`] = `
<div>
  Welcome to our slice of paradise
</div>
`;
